### PR TITLE
Feature/fix layout and margins

### PR DESCRIPTION
**Pull request by draude**

### CODE REVIEW

**Review by kiriamcf-revo — APPROVED** [review]
LGTM <3

File: resources/views/filters/applied.blade.php [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
-    @foreach($report->availableFilters() as $field)
-        @if($report->filters->isFilteringBy($field->getFilterField()))
-            @php($count++)
+@php($fields = [])
+@foreach($report->availableFilters() as $field)
+    @php($filterField = $field->getFilterField())
+    @if($report->filters->isFilteringBy($filterField))
+        @php($fields[] = $field)
+    @endif
+@endforeach
```
Comment (BadChoice) [inline resources/views/filters/applied.blade.php:7]: tens un before/after de com queda?

Comment (draude) [reply resources/views/filters/applied.blade.php:7]: @php($fields = [])
@foreach($report->availableFilters() as $field)
    @php($filterField = $field->getFilterField())
    @if($report->filters->isFilteringBy($filterField))
        @php($fields[] = $field)
    @endif
@endforeach

@if(count($fields) > 0)
    <div id="sidecar-applied-filters" class="mt-4 flex flex-wrap items-center gap-1">
        @foreach($fields as $field)
            @if ($field instanceof \Revo\Sidecar\ExportFields\Boolean)
                @include('sidecar::filters.applied.boolean')
            @elseif ($field instanceof \Revo\Sidecar\ExportFields\Number)
                @include('sidecar::filters.applied.number')
            @elseif ($field instanceof \Revo\Sidecar\ExportFields\Text)
                @include('sidecar::filters.applied.text')
            @elseif ($field instanceof \Revo\Sidecar\ExportFields\DateTime)
                @include('sidecar::filters.applied.time')
            @else
                @include('sidecar::filters.applied.select')
            @endif
        @endforeach
        <a href="{{ url(request()->getPathInfo()) }}">
            <x-ui::tertiary-button icon="xmark">
                {{__(config('sidecar.translationsPrefix').'clearAll')}}
            </x-ui::tertiary-button>
        </a>
    </div>
@endif

//////////////////////////////////////////////////////////////////////////

<div id="sidecar-applied-filters" class="mt-4 flex flex-wrap items-center gap-1">
    @php($count = 0)
    @foreach($report->availableFilters() as $field)
        @if($report->filters->isFilteringBy($field->getFilterField()))
            @php($count++)
            @if ($field instanceof \Revo\Sidecar\ExportFields\Boolean)
                @include('sidecar::filters.applied.boolean')
            @elseif ($field instanceof \Revo\Sidecar\ExportFields\Number)
                @include('sidecar::filters.applied.number')
            @elseif ($field instanceof \Revo\Sidecar\ExportFields\Text)
                @include('sidecar::filters.applied.text')
            @elseif ($field instanceof \Revo\Sidecar\ExportFields\DateTime)
                @include('sidecar::filters.applied.time')
            @else
                @include('sidecar::filters.applied.select')
            @endif
        @endif
    @endforeach
    @if($count > 0)
            <a href="{{ url(request()->getPathInfo()) }}">
                    <x-ui::tertiary-button icon="xmark">
                        {{__(config('sidecar.translationsPrefix').'clearAll')}}
                    </x-ui::tertiary-button>
            </a>
    @endif
</div>

Comment (kiriamcf-revo) [reply resources/views/filters/applied.blade.php:7]: Em sembla que vol dir amb imatges

Comment (draude) [reply resources/views/filters/applied.blade.php:7]: Captura de pantalla 2024-10-18 a les 11 51 03

Captura de pantalla 2024-10-18 a les 11 54 18

**Review by BadChoice — APPROVED** [review]

**Pull request merged**